### ISSUE TITLE
When I use google provider for authorization login, I get a timeout locally  

### ISSUE THREAD

**Issue by yikejason** [adrianhajdin/project_next_13_ai_prompt_sharing#79]
<img width="1121" alt="WechatIMG19341" src="https://github.com/adrianhajdin/project_next_13_ai_prompt_sharing/assets/22877578/a78fdbf1-d9b2-48bb-a28f-4b3969762e02">
<img width="1432" alt="image" src="https://github.com/adrianhajdin/project_next_13_ai_prompt_sharing/assets/22877578/92a10579-14de-4b2c-969a-48d5284ac001">

can you help me soloving this issue  please.    Thank you

**Comment by aabu960** [comment]
I'm getting similar error.

**Comment by aabu960** [comment]
I fixed mine. the problem is comming from your  computer time & date kindly change the time to 1hr ahead of your local time. 

if it doesnt work change callbacks to callback in the [...nexauth] route.js.

**Comment by yikejason** [comment]
> I fixed mine. the problem is comming from your computer time & date kindly change the time to 1hr ahead of your local time.
> 
> if it doesnt work change callbacks to callback in the [...nexauth] route.js.

Yeah  thank you for answering.  I have already sloved it.   It works well now.   When I deployed it on Vercel,  it works well
